config/config.txt

```diff
@@ -571,8 +571,8 @@ AUTO_PROFILE
 RESPAWN_TIMER 600
 
 ## Custom shuttle spam prevention. Changine these numbers allows you to change the maxsize and amount of custom shuttles.
-MAX_SHUTTLE_COUNT 20
-MAX_SHUTTLE_SIZE 300
+MAX_SHUTTLE_COUNT 999999
+MAX_SHUTTLE_SIZE 9999999
```

Comment:
good god why

Reply:
T H E large shuttle

Reply:
this is gonna crash because of a funny guy on live but aight